services/ops/TopicWatcher/README.md

```diff
@@ -0,0 +1,52 @@
+## Topic Watcher Agent
+
+The Topic Watcher Agent listens to a set of configured topics and publishes an alert if they are not published within 
+some time limit.  In addition to “standard” topics the Topic Watcher Agent supports inspecting device "all" topics. This
```

Comment:
Is `"standard" topics` defined somewhere else? if not a parenthetical of what standard means would be useful.